index.html

```diff
@@ -31,11 +31,12 @@ <h3>Advanced algorithms plainly explained</h3>
       </nav>
     </header>
     <main>
-      Welcome to The Algorithmist! There will be content here soon.
+      <p>Welcome to The Algorithmist! There will be content here soon.</p>
       <div class="post">
-        <h2>Aho-Corasick: Fast, Simultaneous Multi-String Search</h2>
-        <p>Something something content preview content preview</p>
-        <a href="/algos/ahocorasick">continue</a>
+        <div>
+          <h2><a href="/ahocorasick">Aho-Corasick</a></h2>
```

Comment:
I accidentally typo-ed the URL here, but we can fix it once the page is created.
```suggestion
          <h2><a href="/algos/ahocorasick">Aho-Corasick</a></h2>
```